src/openzaak/components/documenten/query/cmis.py

```diff
@@ -1119,7 +1126,19 @@ def format_zaak_and_zaaktype_data(
 
         if zaak:
             return format_zaak_and_zaaktype_data(zaak)
-
+    elif verzoek_url:
+        client = Service.get_client(verzoek_url)
+        objectverzoeken = client.list("objectverzoek", {"verzoek": verzoek_url})
+        zaakverzoeken = [
+            objectverzoek
+            for objectverzoek in objectverzoeken
+            if objectverzoek["objectType"] == "zaak"
+        ]
+        if zaakverzoeken:
+            # FIXME what if there is more than one objectverzoek? Is this even possible?
```

Comment:
this is where I'd expect a link to the standard :P

Reply:
I can't seem to find anything about it in https://github.com/VNG-Realisatie/gemma-zaken/blob/master/docs/_content/standaard/zaken/index.md or https://github.com/VNG-Realisatie/gemma-zaken/blob/master/docs/_content/standaard/verzoeken/index.md.

In the code there doesn't seem to be a restriction on this either

Reply:
I think we should seek clarification then - let's create an issue of the type "question" in the gemma-zaken repo.

Reply:
https://github.com/VNG-Realisatie/gemma-zaken/issues/2071

Reply:
@SilviaAmAm do you know why the zaak information is required for Alfresco when creating an ObjectInformatieObject?

Reply:
I see in the cmis-adapter code that the document is then moved/copied to the related zaak folder so that in the file browser (in Alfresco for example), the document is listed with the zaak.

I think this isn't exactly correct for verzoeken, as the zaak is only tangentially related to the zaak. It would probably make more sense to have a directory structure for a verzoek en put the document there:

1. The verzoek is the primary object
2. There's no guarantee the verzoek is related to a zaak
3. One verzoek can lead to multiple zaken
4. The zaken can be created later _because of_ the verzoek, possibly through human action (as opposed to an automated system)

The relevant code is in `drc_cmis.client.CMISClient.create_oio`